bin/README.md

```diff
@@ -19,6 +19,8 @@ If help required - ask BSP team.
 #### Sample command line
 
 ```bash
+$ export SAS_TOKEN="sv=2019-02-02&spr=https%2Chttp&se=2019-12-10T15%3A45%3A13Z&sr=c&sp=wl&sig=OmUS7%2BH62ah1rdQr0r36bkA0EZ10GH6fNSP54NAL0Lw%3D"
```

Comment:
& is missing here ?

Reply:
The URL is `"http://bulkscan$ENVI.blob.core.windows.net/$CONTAINER/$ZIP_FILE_NAME?$SAS_TOKEN"``